src/main/java/com/cburch/logisim/Main.java

```diff
@@ -93,7 +93,8 @@ public static void main(String[] args) {
   public static final LogisimVersion VERSION = new LogisimVersion(3, 5, 0);
   public static final int COPYRIGHT_YEAR = 2021;
   public static final String APP_DISPLAY_NAME = APP_NAME + " v" + VERSION;
-  public static final String APP_URL = "https://github.com/logisim-evolution/";
+  public static final String APP_URL = "https://github.com/logisim-evolution/logisim-evolution";
+  public static final String ORG_URL = "https://github.com/logisim-evolution/";
```

Comment:
Can you use `APP_URL` instead `ORG_URL`? The main intent I made `APP_URL` pointing to the organization is to make it shorter. It also avoids, confusing to some, "duplicated" `logisim-evolution` part. IMHO this brings more value than pointing to project page directly.

Reply:
Pointing to the organization is no problem at the moment. But as soon as the organization has multiple repositories, it will be more irritating than having `logisim-evolution` twice in the URL.
And if length is a concern, you can simply use `ORG_URL`.

Reply:
You are "solving" non existing problem here. There's no multiple repositories. There's no point for this change. It just introduces another, almost redundant constant for no real benefit.